apps/entry/views.py

```diff
@@ -282,6 +282,21 @@ def unverify_entry(self, request, **kwargs):
         )
 
 
+class EntryListView(generics.GenericAPIView):
+    serializer_class = EntrySerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get_queryset(self):
+        return Entry.objects.all()
+
+    def post(self, request, version=None):
+        entries_id = request.data['ids']
+        entries = Entry.objects.filter(id__in=entries_id)
+        serializer_class = self.get_serializer_class()
+        serializer = serializer_class(entries, many=True, context={'request': request})
+        return response.Response(serializer.data)
+
+
```

Comment:
No No No. As specified before let's only add a filter to the already existing entry API. So that user can filter by entry id as well.

Also, make sure users only have access to the entry they should.

Let's add a custom filter to entryfilterset, then add a test where it should filter using entry id. Also, add another test where user ask for non-authorized entry as well.

Reply:
Makes more sense!